R/dgp-lib.R

```diff
@@ -326,11 +296,11 @@ correlated_linear_gaussian_dgp <- function(n, p_uncorr, p_corr,
   }
   
   # simulate betas_corr and betas_uncorr
-  betas_corr <- generate_coef(betas = betas_corr, p = p_corr, s = s_corr,
```

Comment:
Should we use `...` in the same way as in `linear_gaussian_dgp` for consistency? If so, we should remove `betas_corr_sd` and `betas_uncorr_sd` from the function arguments. Similar question for all other dgp functions in this script.

Reply:
Ya, that's the plan. Just focused on getting things right for `linear_gaussian_dgp()` for now, then will update the others as well.

Reply:
Great!